Objects/descrobject.c

```diff
@@ -130,7 +130,11 @@ classmethod_get(PyObject *self, PyObject *obj, PyObject *type)
     if (descr->d_method->ml_flags & METH_METHOD) {
         cls = descr->d_common.d_type;
     }
-    return PyCMethod_New(descr->d_method, type, NULL, cls);
+    PyObject *mod = PyObject_GetAttr((PyObject*)type, &_Py_ID(__module__));
```

Comment:
This will add an overhead for use of class methods. We should try to find other solution.

Reply:
> overhead for use of class methods

You meant for *adding* static/class methods?

> We should try to find other solution.

Unlikely e.g. ``__name__``, it's a member of PyCFunctionObject.  I think we can set one only in PyCMethod_New and there is no more cheapest solution.